os/arch/arm/src/armv7-m/up_hardfault.c

```diff
@@ -85,7 +85,9 @@
 
```

Comment:
Kindly mention start and end commit ID's in the commit message as well as the PR description

Reply:
Along with the branch names*